src/modules/generateRandomNumber.js

```diff
@@ -9,7 +1,19 @@
+/* eslint-disable no-console */
+/* eslint-disable padding-line-between-statements */
 function generateRandomNumber() {
-  /* Write your code here */
+  let randomNumber;
+
+  do {
+    randomNumber = Math.floor(Math.random() * 9000) + 1000;
+  } while (new Set(randomNumber.toString()).size !== 4);
+
+  // eslint-disable-next-line no-console
+  console.log('Generated Random Number:', randomNumber);
+
+  return randomNumber;
 }
 
-module.exports = {
-  generateRandomNumber,
-};
+const newRandomNumber = generateRandomNumber();
+console.log('Random Number from Function:', newRandomNumber);
```

Comment:
This console log statement is also for debugging. Ensure it's necessary for your application, or remove it if it's not needed.